core/templates/dev/head/pages/exploration_player/learner_local_nav.html

```diff
@@ -7,24 +7,19 @@
     </a>
   </li>
   <li>
-    {% if can_edit %}
-      <a ng-href="/create/<[explorationId]>" uib-tooltip="<['I18N_PLAYER_EDIT_TOOLTIP' | translate]>" tooltip-placement="bottom" target="_blank">
-        <i class="material-icons">&#xE254;</i>
-        <span class="oppia-icon-accessibility-label"><['I18N_PLAYER_EDIT_TOOLTIP' | translate]></span>
-      </a>
-    {% elif username %}
-      <a class="protractor-test-exploration-suggestion-popup-link" ng-click="showLearnerSuggestionModal()" uib-tooltip="Suggest Changes" tooltip-placement="bottom" target="_blank">
-        <i class="material-icons">&#xE254;</i>
-        <span class="oppia-icon-accessibility-label">Suggest Changes</span>
-      </a>
-    {% endif %}
+    <a ng-if="canEdit" ng-href="/create/<[explorationId]>" uib-tooltip="<['I18N_PLAYER_EDIT_TOOLTIP' | translate]>" tooltip-placement="bottom" target="_blank">
+      <i class="material-icons">&#xE254;</i>
+      <span class="oppia-icon-accessibility-label"><['I18N_PLAYER_EDIT_TOOLTIP' | translate]></span>
+    </a>
+    <a ng-if="username" class="protractor-test-exploration-suggestion-popup-link" ng-click="showLearnerSuggestionModal()" uib-tooltip="Suggest Changes" tooltip-placement="bottom" target="_blank">
```

Comment:
@WickedBrat, I think both the conditions should be included here i.e. ng-if="!can_edit && username".